cpp/src/parquet/types.cc

```diff
@@ -69,11 +69,6 @@ std::unique_ptr<Codec> GetCodec(Compression::type codec, int compression_level)
     throw ParquetException(ss.str());
   }
 
-  if (codec == Compression::LZ4) {
```

Comment:
it this right?  how are we going about deprecating LZ4 vs LZ4 RAW or is that  different issue/PR?

Reply:
We're not deprecating anything, `Compression::LZ4` simply means "use the LZ4 raw (block) format" as it was always meant to be. We're unable to reliably produce Hadoop-readable files anywhere, it would be counter-productive to still produce them by default.